### PR TITLE
test: smoke tests fixed

### PR DESCRIPTION
**Pull request by g4s8**
Fixed runner script, fail on error

Ticket: #847

### CODE REVIEW

File: examples/run.sh [hunk 1]
```diff
@@ -145,25 +144,36 @@ function run_test {
...
+  fi
 }
 
 create_network
 create_volume
 start_artipie
 
-declare -a tests=(binary debian docker go helm maven npm nuget php rpm conda)
+if [[ -z "$1" ]]; then
+  declare -a tests=(binary debian docker go helm maven npm nuget php rpm)
```
Comment (olenagerasimova) [inline examples/run.sh:160]: Why conda is not in the list?

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@g4s8 one question, take a look please

**Comment by olenagerasimova** [comment]
@g4s8 please, resolve conflicts and we are good to merge

**Comment by g4s8** [comment]
@olenagerasimova thanks, resolved

**Review by olenagerasimova — APPROVED** [review]

**Pull request merged**